setup.py

```diff
@@ -99,23 +99,22 @@ def get_cryptography_version():
         "pytz",
         "tzlocal",
         "aiolancium",
-    ],
+    ]
+    + REST_REQUIRES,
     extras_require={
         "docs": [
             "sphinx",
             "sphinx_rtd_theme",
             "sphinxcontrib-contentui",
             "myst_parser",
         ],
-        "rest": REST_REQUIRES,
         "test": TESTS_REQUIRE,
         "contrib": [
             "flake8",
             "flake8-bugbear",
             "black; implementation_name=='cpython'",
         ]
-        + TESTS_REQUIRE
-        + REST_REQUIRES,
+        + TESTS_REQUIRE,
```

Comment:
Same as above, include `*TESTS_REQUIRE` into the list instead of concatenating `+ TESTS_REQUIRE`. (I cannot make a code suggestion here, fittingly because the diff is too messy.)

Reply:
Fine with me and fixed!